test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/SaveTempDataFilter/SaveTempDataPropertyFilterTestBase.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Internal
+{
+    public class SaveTempDataPropertyFilterTestBase
+    {
+        public static IList<TempDataProperty> BuildTempDataProperties<TSubject>()
```

Comment:
Could we not have this type? It's used in 4 places and we can construct the `TempDataProperty` manually in a helper method sans reflection